src/library.js

```diff
@@ -85,8 +86,14 @@ LibraryManager.library = {
       return -1;
     }
   },
+#else
+  $setFileTime: function(path, time) {
+    // No filesystem support; no-op.
```

Comment:
Perhaps returning `setErrNo(ENOENT); return -1` would make more sense?

Reply:
I considered that, but it seems better to not error given that there are no files, really, rather than always error. That is, the problem cannot be noticed - this is "write only memory" as the joke goes. So all it could do is confuse a program that cares about the return value.

Reply:
I think I disagree... if the file doesn't exist we should return an error.   I can't imagine a program that break because isn't set utime on a non-existent file.    `stat` and `open` will both fail.. so why allow utimes to succeed?

Reply:
Hmm, fair enough. While there is no way to observe if the file exists or not, however, since file support is disabled, but I guess only the standard streams exist... which I suppose one cannot utime?